src/Compilers/CSharp/Test/Semantic/Semantics/OverloadResolutionPerfTests.cs

```diff
@@ -656,5 +656,60 @@ class C
             var comp = CreateCompilation(sourceBuilder.ToString());
             comp.VerifyDiagnostics();
         }
+
+        [ConditionalFact(typeof(IsRelease))]
+        public void DefiniteAssignment_ManySwitchCasesAndLabels()
```

Comment:
On my machine, this test takes ~4 mins before the change, 22 secs after the change.